docs/api/features/placement.md

```diff
@@ -1,3 +1,137 @@
 # Placement
 
-What are the default rules in SDC, do they differ in hosted Triton?
+When using Docker with a Triton data center (a.k.a. SDC) as your Docker Host,
+your Docker containers can be placed on any physical server in the data center.
+This is one of the basic benefits of Triton's Docker solution: with just the
+vanilla `docker` client you can get the benefit of spreading your containers
+across multiple servers, setting the groundwork for high availability (HA)
+services and avoiding single points of failure (SPOF).
+
+This document explains how Triton places new containers on servers and what
+facilities are exposed for controlling placement.
+
+
+## Default placement
+
+By default, Triton attempts to spread all containers (and non-Docker containers
+and VMs) owned by a single account across separate physical servers.
```

Comment:
Hrm only east3b uses random spread strategy. The rest of JPC is still on min memory.